.github/workflows/clingo.yml

```diff
@@ -51,92 +53,70 @@ jobs:
       - name: Install clingo-bootstrap
         run: |
           pip install --upgrade pip
-          brew install bison
           export PATH="$(brew --prefix bison):$PATH"
           . spack/share/spack/setup-env.sh
-          spack external find --not-buildable cmake bison
+
+          rm -rf $(spack location -p pod man)
```

Comment:
I'm not sure what this is trying to accomplish, but at the very least it looks like a typo `pod man` vs `podman`.